docs/tutorials/mongoid-upgrade.txt

```diff
@@ -362,6 +362,25 @@ Example Mongoid 7.1 behavior:
 Mongoid provides :ref:`sharding management Rake tasks <sharding-management>`
 to shard collections according to shard keys declared in models.
 
+Query Cache Moved to Driver
+------------------------------------
+
+Minor change: The Mongoid query cache has been deprecated in favor of a new
```

Comment:
Similarly to the other PR, I suggest starting with the fact that Mongoid will use the driver's QC in 2.14, then listing the benefits, then stating the legacy QC is deprecated, and using the term  "the legacy Mongoid query cache" to refer to it.

Also can this link to the QC section in the docs (will potentially need an anchor added to the section so that it can be linked to).